File: adwin/src/main/java/com/bharani/adwin/utils/ADBucket.java
package com.bharani.adwin.utils;

public class ADBucket {

    private int capacity;
    private double content;

    ADBucket(int capacity,double content){
        this.capacity = capacity;
        this.content = content;
    }

    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public double getContent() {
        return content;
    }

    public void setContent(double content) {
        this.content = content;
    }

    public void incCapacity(){
        this.capacity++;
    }

}

File: algorithm/AdBucket.py


class ADBucket(object):
    """docstring for ADBucket."""

    def __init__(self, capacity=0, content=0.0):
        super(ADBucket, self).__init__()
        self.capacity = capacity
        self.content = content

    def incCapacity(self):
        self.capacity+=1
        
